courses/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Tests should generally go in the same PR which introduces the functionality to be tested, otherwise there's a high probability they get prioritized lower or just forgotten about. There isn't much to test yet but I'd like to see at least tests for `JsonListField` and some tests that do something basic with the API